.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,40 @@
 <!--
-Prepend your PR title with one of the following:
-- `feat:` A new feature
-- `fix:` A bug fix
-- `docs:` Documentation only changes
-- `style:` Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-- `refactor:` A code change that neither fixes a bug or adds a feature
-- `perf:` A code change that improves performance
-- `test:` Add missing tests
-- `chore:` Changes to the build process or auxiliary tools and libraries such as documentation generation
+🤖 This repo uses Conventional Commits (conventionalcommits.org) to automate
+release notes and versioning. Title your PR using the following template:
+
+<type>(<scope>): <subject>
+
+Types:
+- feat: new feature for the user, not a new feature for build script
+- fix: bug fix for the user, not a fix to a build script
+- docs: changes to the documentation
+- build: changes that affect the build system or external dependencies
+- test: adding missing tests, refactoring tests; no production code change
+- refactor: refactoring production code, eg. renaming a variable
+- style: formatting, missing semi colons, etc; no production code change
+- chore: updating grunt tasks etc; no production code change
+- revert: reverts a previous commit
+- perf: changes that improve performance
+- ci: changes to CI configuration files and scripts (eg. GitHub Actions)
+
+Examples:
+- feat(button): primary variant
+- fix(action-bar): inherit event-listeners
 -->
 
-**Describe the problem prior to this PR (link ticket/issue):**
+## Describe the problem this PR addresses
+<!--
+🤐 If you are a Square employee, be mindful of any internal information
+you share in this public repository.
+-->
 
-**Describe the changes in this PR:**
-<!-- Tip: inline screenshots to better communicate the changes -->
+## Describe the changes in this PR
+<!--
+📸 Inline screenshots to better communicate the changes
```

Comment:
Not sure if this is really needed since styleguides get auto-deployed for every PR, but I guess they could still copy&paste the link from the bot comment into their PR description.

Reply:
I think eventually the auto-deploys should expire/be cleaned.

Also something to think about if the assets are being pushed to the same repo is that it weights down the entire repo history. Each deploy will get slower and slower.

Reply:
ah yeah, I made an issue for that https://github.com/square/maker/issues/45

Also, I believe the history depth can be controlled when checking out a git branch/repo, I can't remember if I made that optimization in the workflow already but it should be possible to add so that the workflow doesn't get progressively slower when more commits are added to the `deploys` branch.